src/main/java/com/uid2/admin/vertx/service/SharingService.java

```diff
@@ -120,7 +120,7 @@ private JsonObject jsonFullKeyset(Keyset keyset) {
         jo.put("keyset_id", keyset.getKeysetId());
         jo.put("site_id", keyset.getSiteId());
         jo.put("name", keyset.getName());
-        jo.put("allowlist", keyset.getAllowedSites());
+        jo.put("allowed_sites", keyset.getAllowedSites());
```

Comment:
we allow ID_READER to access all keysets whose 'allowed_site' equals to 'null'. I wonder when you're going to set it to null in Admin.

Reply:
Keep in mind this is old code, only the field name is being changed. Looking at https://admin-integ.uidapi.com/adm/keysets.html , most allowlists are null.